chef/cookbooks/provisioner/templates/default/crowbar_join.suse.sh.erb

```diff
@@ -134,8 +134,10 @@ sync_time() {
 
 <% if @target_platform_version.to_f < 12.0 %>
     SNTP_OPTS="-P no -r"
-<% else %>
+<% elsif @target_platform_version.to_f == 12.0 %>
```

Comment:
I wasn't sure whether 12 can be represented or not. in any case "12.0 < 12.1" should be true no matter which IEEE rounding mode or precision error is in effect, and thats all that matters here. we have not and will not release SLES 12.0999999999999999 for that matter. And Float::EPSILON is pretty small (and in IEEE rounding, 12.1-3*Float::EPSILON is still 12.1)

12.09 is slightly less readable though.